pkg/services/ngalert/state/template/template_test.go

```diff
@@ -48,7 +48,7 @@ func TestValueString(t *testing.T) {
 	}, {
 		name:     "1.0 is returned as integer value",
 		value:    Value{Value: 1.0},
-		expected: "1",
+		expected: "1.0",
```

Comment:
Since we would now require two decimal places the value expected here is 1.0

Reply:
I wonder if we can improve the logic a bit and keep the number integer.

Reply:
Yeah lets modify the function to return an integer value

Reply:
Now it works without modifying the tests.